matlab/CARFAC_binaural.m

```diff
@@ -24,36 +24,32 @@
 
 tic
 
-file_signal = wavread('../test_data/binaural_test.wav');
+file_signal = audioread('../test_data/binaural_test.wav');
 file_signal = file_signal(9000+(1:15000));  % trim for a faster test
 
 itd_offset = 22;  % about 1 ms
 test_signal = [file_signal((itd_offset+1):end), ...
                file_signal(1:(end-itd_offset))] / 10;
-             
-CF_struct = CARFAC_Design;  % default design
-
 % Run stereo test:
-n_ears = 2
-CF_struct = CARFAC_Init(CF_struct, n_ears);
+n_ears = 2;             
+CF_struct = CARFAC_Design(n_ears);  % default design with 2 ears
+CF_struct = CARFAC_Init(CF_struct);
   
 [CF_struct, nap_decim, nap] = CARFAC_Run(CF_struct, test_signal, agc_plot_fig_num);
 
 % Display results for 2 ears:
 for ear = 1:n_ears
   smooth_nap = nap_decim(:, :, ear);
   figure(ear + n_ears)  % Makes figures 3 and 4
-  image(63 * ((smooth_nap)' .^ 0.5))
+  image(63 * abs((smooth_nap)' .^ 0.5))
```

Comment:
Why is the abs needed here?  Is the nap not nonnegative?

Reply:
Smooth_nap is complex, and this is (no-longer?) supported by matlabs image call

Reply:
![image](https://user-images.githubusercontent.com/25512064/109629018-4f615d80-7b3b-11eb-812a-62ee7a1a3439.png)
heres a screeny of the error matlab throws

Reply:
![image](https://user-images.githubusercontent.com/25512064/109630014-589efa00-7b3c-11eb-942b-b4b8665f772a.png)

Reply:
I need to dig a little deeper, but Im guessing for the meantime its probably ok to rectify negative nap values? 
We aren't modelling actual neuron firing action potentials.